bigbluebutton-html5/imports/ui/components/app/styles.scss

```diff
@@ -152,10 +152,37 @@ $bars-padding: $lg-padding-x - .45rem; // -.45 so user-list and chat title is al
   flex: 1 100%;
   order: 2;
 
+  flex-direction: row;
```

Comment:
If you are adding another element inside media (closed-captions), than you should change the "order" for the first and second elements. I think the first element would be this:
https://github.com/JaeeunCho/bigbluebutton/blob/87b013fd5b57aa8f247a38974268516ef7449281/bigbluebutton-html5/imports/ui/components/media/component.jsx#L36
Second would be closed-captions box.